client/core/core.go

```diff
@@ -2561,6 +2561,7 @@ func (c *Core) DiscoverAccount(dexAddr string, appPW []byte, certI interface{})
 			atomic.StoreUint32(&dc.reportingConnects, 1)
 			c.wg.Add(1)
 			go c.listen(dc)
+			go dc.subPriceFeed()
```

Comment:
On second though, I wonder if `listen` should just do `dc.subPriceFeed`.

Reply:
It looks like `dc.subPriceFeed` is always called after `listen` so that makes sense.